docs/interfaces.md

```diff
@@ -365,3 +365,9 @@ Auto-Connect: no
 Can manage timeservers directly separate from ``config core``.
 
 * Auto-Connect: no
+
+### kernel-module-control
+
+Allow to insert kernel modules.
+
+* Auto-Connect: no
```

Comment:
Can you insert this alphabetically in the 'Advanced' section? Also, please rephrase to be more like the others: 'Can insert kernel modules. This interface gives privileged access to the device.'